tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -294,6 +294,21 @@ impl<F, T> Format<F, T> {
     /// See [`Pretty`].
     ///
     /// Note that this requires the "ansi" feature to be enabled.
+    ///
+    /// # Options
+    ///
+    /// [`Format::with_ansi`] can be used to disable ANSI formatting in event formatting. However,
```

Comment:
nit/TIOLI: I might say "ANSI terminal escape codes (which enable formatting such as colors, bold, italic, etc)" or something. i've noticed that in the past, some people are confused when the docs just say "enables ANSI" or whatever without describing what that actually is...

Reply:
Fixed in 45f21dc :sparkles: